src/main/java/com/xpanxion/java/springboot/da1/demo/controller/student5/WorkoutHistoryController5.java

```diff
@@ -30,4 +28,9 @@ public WorkoutHistory5 checkout(@PathVariable("memberId") Long memberId,
         return workoutHistoryService.workoutCheckOut(memberId,
                 new SimpleDateFormat("yyyy-MM-dd HH:mm").parse(time));
     }
+
+    @GetMapping("student5/api/v1/member/{memberId}/workout-history")
+    public List<WorkoutHistory5> workoutHistory(@PathVariable("memberId") Long memberId) {
+        return workoutHistoryService.findAllByMemberMemberId(memberId);
```

Comment:
Does this match the output required by the assignment?  I'm looking for something like:

```
[
    {
        "memberId": 32,
        "timeUtc": "2022-01-02T19:48:28.000+00:00",
        "checkType": "CHECK_IN"
    },
    {
        "memberId": 32,
        "timeUtc": "2022-01-02T19:48:28.000+00:00",
        "checkType": "CHECK_OUT"
    },
    {
        "memberId": 32,
        "timeUtc": "2022-01-02T14:48:28.000+00:00",
        "checkType": "CHECK_IN"
    },
    {
        "memberId": 32,
        "timeUtc": "2022-01-02T15:48:28.000+00:00",
        "checkType": "CHECK_OUT"
    }
]
```

Reply:
I'm not sure if this is close enough.